src/assets/scripts/math/circle.js

```diff
@@ -64,3 +64,27 @@ export const fix_angle = (radians) => {
 
     return radians;
 };
+
+/**
+ * normalize angles to within 0° - 360°
+ * @param  {number} degrees an angle
+ * @return {number}         an angle within [0,360]
+ */
+export const degrees_normalize = (degrees) => {
+    if (degrees < 0) {
+        return 360 + (degrees % 360);
+    }
+    else return degrees % 360;
+};
+
+/**
+ * normalize angles to within 0 - 2π
+ * @param  {number} radians an angle
+ * @return {number}         an angle within [0,2π]
+ */
+export const radians_normalize = (radians) => {
+    if (radians < 0) {
+        return (Math.PI*2) + (radians % (Math.PI*2));
+    }
+    else return radians % (Math.PI*2);
```

Comment:
If the latter is true, then I must have just done something stupid like spelled it wrong, or forgot to re-transpile... oops! I'll put it back to calling `tau()` if it does just work.